### PR TITLE
Update to Indra 2

### PR DESCRIPTION
**Pull request by broccolai**
Draft while Indra 2 is a snapshot, this also lets us update to Cadix licenser 0.6 proper.
Builds complete in terminal but IntelliJ sync has some issues, also not sure how to fix the todo in the Fabric build file

### CODE REVIEW

File: cloud-minecraft/cloud-fabric/build.gradle.kts [hunk 1]
```diff
@@ -75,7 +75,8 @@ dependencies {
 }
 
 indra {
-    includeJavaSoftwareComponentInPublications.set(false)
+// TODO: RE-ADD
+//    includeJavaSoftwareComponentInPublications.set(false)
```
Comment (zml2008) [inline cloud-minecraft/cloud-fabric/build.gradle.kts:79]: includeJavaSoftwareComponentInPublications(false)

**Review by jpenilla — APPROVED** [review]

File: build.gradle.kts [hunk 2]
```diff
@@ -13,18 +12,18 @@ import org.gradle.api.plugins.JavaPlugin.COMPILE_ONLY_API_CONFIGURATION_NAME
 import org.gradle.api.plugins.JavaPlugin.TEST_IMPLEMENTATION_CONFIGURATION_NAME
 
 plugins {
-    val indraVersion = "1.3.1"
+    val indraVersion = "2.0.0"
     id("net.kyori.indra") version indraVersion apply false
     id("net.kyori.indra.checkstyle") version indraVersion apply false
-    id("net.kyori.indra.publishing.sonatype") version indraVersion apply false
+    id("net.kyori.indra.publishing.sonatype") version indraVersion
```
Comment (jpenilla) [inline build.gradle.kts:18]: why was this changed?

File: build.gradle.kts [hunk 3]
```diff
@@ -13,18 +12,18 @@ import org.gradle.api.plugins.JavaPlugin.COMPILE_ONLY_API_CONFIGURATION_NAME
...
     id("net.kyori.indra.license-header") version indraVersion apply false
-    id("org.cadixdev.licenser") version "0.6.0-SNAPSHOT" apply false
+    id("org.cadixdev.licenser") version "0.6.0" apply false
```
Comment (jpenilla) [inline build.gradle.kts:20]: this line can just be removed.

**Review by jpenilla — CHANGES_REQUESTED** [review]

File: build.gradle.kts (hunk 2 above)
Comment (broccolai) [reply build.gradle.kts:18]: gradle publishing plugin requires being applied to the root project

**Review by jpenilla — APPROVED** [review]
